crates/sui-adapter/src/temporary_store.rs

```diff
@@ -142,7 +142,7 @@ impl<S> TemporaryStore<S> {
         gas_object.storage_rebate = gas_status.charge_storage_mutation(
             gas_object_size,
             gas_object_size,
-            gas_object.storage_rebate,
+            gas_object.storage_rebate.into(),
```

Comment:
What are the `into`s to/from?

Reply:
u64 into SuiGas
See earlier comment about strict units and needing reworking